<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoCatcher</class>
 <widget class="QWidget" name="VideoCatcher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1601</width>
    <height>776</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>视频录制</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>0</y>
     <width>1391</width>
     <height>641</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="selectLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>选择摄像头：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="selectCamera">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>自带摄像头</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>外接摄像头</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="openCamera">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>打开摄像头</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startVideo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>开始录像</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopVideo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>400</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>停止录像</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeCamera">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭摄像头</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
